meta-resin-common/recipes-containers/docker-disk/docker-utils.inc

```diff
@@ -1,19 +1,23 @@
-RESIN_CHECK_CONN_URL ?= "index.docker.io"
+RESIN_CHECK_CONN_URL ?= "http://index.docker.io"
```

Comment:
Currently this goes all the way with redirects to `https://hub.docker.com/ `, would that be better to use that? 

```
$ curl -I http://index.docker.io
HTTP/1.1 301 Moved Permanently
Location: https://index.docker.io/

$ curl -I https://index.docker.io
HTTP/1.1 301 MOVED PERMANENTLY
Location: https://registry.hub.docker.com/

$ curl -I https://registry.hub.docker.com/
HTTP/1.1 301 MOVED PERMANENTLY
Location: https://hub.docker.com/

$ curl -I https://hub.docker.com/ 
HTTP/1.1 200 OK
```